docs/tutorial.md

```diff
@@ -2,593 +2,253 @@
 
-#### Node Package manager
+![Our bot saying "Hello there!"](https://i.imgur.com/Wz0ia8w.png)
 
-The last thing you need to know is how to use the [Node Package Manager](https://www.npmjs.com/).  
-NPM is automatically installed when you install Node.  
-NPM is used to get useful packages that other people created that can do useful things for you.  
-You can search for packages on [their website](https://www.npmjs.com/), and then install them using the `npm install` command in your terminal.  
-To install Mineflayer for example, run `npm install mineflayer`  
-
-Then, Node can access installed modules by using the `require()` function.
+Alternatively, if we want to listen to an event once, we can use `bot.once()`.
 
 ```js
-const mineflayer = require('mineflayer')
+bot.once('chat', () => {
+  bot.chat('I will only listen to this event once!')
+})
 ```
 
-After this, the `mineflayer` variable can be used to access all the features of Mineflayer.
+![Our bot only replies once to .once() event listeners.](https://i.imgur.com/qow3N0f.png)
 
-### Creating a bot
+## Plugins
 
-Now that you know the basics of Javascript, Node and NPM, you're ready to start creating your first bot!  
-If you don't know any of the terms above, you should go back to the [previous section](#javascript-basics)
+Mineflayer lets you use plugins with your bot. This means you can add features to the bot that you (or other people) made. Let's try use PrismarineJS's official `mineflayer-pathfinder` plugin now to make our bot able to easily move to certain locations.
 
-Below is the absolute minimum necessary to create a Mineflayer bot.
+First, we need to install the `mineflayer-pathfinder` package from NPM, which includes our plugin. Go back to your command line and type:
 
-```js
-const mineflayer = require('mineflayer')
-
-const bot = mineflayer.createBot()
+```bash
+npm i mineflayer-pathfinder
 ```
 
-If you run this example, you'll notice that your program will not stop. If you want to stop your currently running program, press `Ctrl` + `c`  
-However, this bot isn't quite useful, as by default this will connect to a Minecraft server running on your machine with the port 25565.  
-If you want to choose which server you want your bot to connect to, you have to pass along a few options.
-
+Now we can import this module in our code like we did before.
 
 ```js
-const mineflayer = require('mineflayer')
-
-const options = {
-  host: 'localhost', // Change this to the ip you want.
-  port: 25565 // Change this to the port you want.
-}
-
-const bot = mineflayer.createBot(options)
+const mineflayerPathfinder = require('mineflayer-pathfinder')
+/* Note: It is good practice to keep all your modules
+         the top of your file. Try to put this under
+         the line you imported Mineflayer.
```

Comment:
Try something like "Make sure to place all your require() imports at the top of the file where possible. This prevents your code from being blocked loading modules at runtime by loading all the dependencies before your program code is ran."

to better explain what's happening